tools/devops/LocProject.json.in

```diff
@@ -22,6 +22,16 @@
 			"LciFile": "",
 			"Parser": "",
 			"LssFiles": []
+		  },
+		  {
+			"SourceFile": "@WORKING_DIRECTORY@/../../src/Resources.resx",
+			"Languages": "",
+			"CopyOption": "LangIDOnName",
+			"OutputPath": "@WORKING_DIRECTORY@/../../src/TranslatedAssemblies",
```

Comment:
Maybe we should put this in src/bgen instead:

```suggestion
			"OutputPath": "@WORKING_DIRECTORY@/../../src/bgen/TranslatedAssemblies",
```

Reply:
I could move that, would you also prefer that the src/Resources.resx be moved to src/bgen/Resources.resx?

Reply:
Yes, please!